File: Exercicio_17/biblioteca/src/entities/Ebook.java
package entities;

public class Ebook extends Livro {
    private double tamanhoArquivo;
    private String formato;

    public Ebook(String titulo, String autor, int anoPublicacao, double tamanhoArquivo, String formato) {
        super(titulo, autor, anoPublicacao);
        this.tamanhoArquivo = tamanhoArquivo;
        this.formato = formato;
    }

    @Override
    public void exibirInfo() {
        super.exibirInfo();
        System.out.println("Tamanho do Arquivo: " + tamanhoArquivo + " MB");
        System.out.println("Formato: " + formato);
    }
}

File: 17_OrientacaoAObjetos/Ebook.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace _17_OrientacaoAObjetos
{
    internal class Ebook : Livro
    {
        public double TamanhoArquivo { get; set; }
        public string Formato { get; set; }

        public override void ExibirInfo()
        {
            Console.WriteLine("\n=== Ebook ===");
            base.ExibirInfo();
            Console.WriteLine("Tamanho do arquivo: " + TamanhoArquivo + "MB");
            Console.WriteLine("Formato: " + Formato);
        }

    }
}
